src/utils/sharedRegex.js

```diff
@@ -0,0 +1,13 @@
+const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]{2,}$/;
+const phoneRegex = /^[\d+()-\s]+$/;
+
+const testRegex = (phoneNumber) => {
```

Comment:
testRegex is too generic, testRegex email ? Or phone? Why not adding a second param for choosing what to test and to cover each case here. Now we have only phone test regex as I can see?